### PR TITLE
updated instructions for libcupti for CUDA Toolkit >= 8.0

### PR DESCRIPTION
**Pull request by kashif**

### CODE REVIEW

**Comment by sb2nov** [comment]
Jenkins, test this please.

File: tensorflow/docs_src/install/install_linux.md [hunk 1]
```diff
@@ -42,9 +42,21 @@ must be installed on your system:
     a list of supported GPU cards.
   * The libcupti-dev library, which is the NVIDIA CUDA Profile Tools Interface.
     This library provides advanced profiling support. To install this library,
-    issue the following command:
+    issue the following command for CUDA Toolkit >= 8.0:
 
     <pre>
+    $ <b>sudo apt-get install cuda-command-line-tools</b>
```
Comment (gunan) [inline tensorflow/docs_src/install/install_linux.md]: Are you sure about this?
On my machine, cuda 7.5 instructions worked for me

Comment (kashif) [reply tensorflow/docs_src/install/install_linux.md]: @gunan the cuda 7.5 instructions pulls in the cuda 7.5 version of libcupti. On Cuda >= 8.0 it doesnt seem to come in its own library, rather it comes in extra withe cuda command line tools deb. So with cuda 8.0 you more or less need to add the library path to the appropriate location. Not sure, does that answer your question?

**Review by gunan — APPROVED** [review]

**Comment by gunan** [comment]
Jenkins, test this please.

**Pull request merged**